app/views/views.py

```diff
@@ -0,0 +1,85 @@
+from flask import Flask,request
+from flask_restplus import Api, Resource, fields
+from ..models.signup import Upload
+
+app = Flask(__name__)
+api = Api(app)
+app.config.from_pyfile('../../config.py')
+signin = api.model('signin',{
+	'fname':fields.String,
+	'lname':fields.String,
+	'email':fields.String,
```

Comment:
I am concerned with how you name your variables fname and lname. I feel they are no readable.